src/libixy-vfio.h

```diff
@@ -0,0 +1,26 @@
+#ifndef LIBIXY_VFIO_H
+#define LIBIXY_VFIO_H
+
+#include <stdint.h>
+
```

Comment:
C++-style comments in the whole file unless multiple lines are needed for a function description